nucypher/policy/conditions/ecdsa.py

```diff
@@ -130,6 +134,7 @@ class ECDSACondition(AccessControlCondition):
     """
 
     CONDITION_TYPE = "ecdsa"  # Add this to ConditionType enum
+    SUPPORTED_CURVES = {c.name: c for c in curves}
```

Comment:
If my comments below are accepted, then you may not need this constant, or it can be moved.